manticore/__init__.py

```diff
@@ -1,3 +1,6 @@
-from .manticore import Manticore, STDIN_INPUT_DEFAULT_SIZE  # noqa
-from .models import variadic  # noqa
-from .utils.helpers import issymbolic  # noqa
+from manticore.utils import log
+
+# Default symbolic input size
+STDIN_INPUT_DEFAULT_SIZE = 256
```

Comment:
This is a bit weird place to have it in, but the point is it needs to be importable from `__main__.py` and from the `manticore/native/manticore.py`.

Reply:
should this be a value adjustable from the `config` system?

Reply:
ideally, yes